### DISCUSSION TITLE
Does it uses a lot of GPU resources?

### DISCUSSION THREAD

**Discussion by toxifiedmatrix** [idiotWu/smooth-scrollbar#339] [Q&A] [answered]
As I have seen the content shakes drastically, with great momentum, you can literally see it vibrating, when I inspect element, and see the website with a zoom of 90% and less. Can you tell something about this?

**Comment by yasserzakaria** [comment]
@idiotWu I have the same issue

**Comment by dolphin-wood** [comment, marked as answer]
This plugin uses `translate3d`, so yes.
